catalog.html

```diff
@@ -56,35 +56,106 @@
     </header>
 
     <main>
-      <ul>
-        <li><a href="">Каталог</a></li>
-        <li><a href="">Инструмент</a></li>
-        <li><a href="">Перфораторы</a></li>
-      </ul>
-
-      <h1>Перфораторы</h1>
-
       <section>
-        <h2>Производители:</h2>
         <ul>
-          <li>BOSCH</li>
-          <li>ИНТЕРСКОЛ</li>
-          <li>MAKIta</li>
-          <li>Dewalt</li>
-          <li>Hitachi</li>
+          <li><a href="">Каталог</a></li>
+          <li><a href="">Инструмент</a></li>
+          <li><a href="">Перфораторы</a></li>
         </ul>
 
-        <h2>Питание:</h2>
+        <h1>Перфораторы:</h1>
+        <h2>СОРТИРОВКА:</h2>
         <ul>
-          <li>Аккумуляторные</li>
-          <li>Сетевые</li>
+          <li>
+            <a href="">ПО ЦЕНЕ</a>
+          </li>
+          <li>
+            <a href="">ПО ТИПУ</a>
+          </li>
+          <li>
+            <a href="">ПО ФУНКЦИОНАЛУ</a>
+          </li>
+          <li>
+            <button type="button">↑</button>
+          </li>
+          <li>
+            <button type="button">↓</button>
+          </li>
         </ul>
 
-        <button type="submit">Показать</button>
+        <h2>фильтр</h2>
+        <form action="https://echo.htmlacademy.ru" method="POST">
+          <fieldset>
+            <legend>ЦЕНА:</legend>
+            <input
+              type="range"
+              name="price"
+              id="price"
+              min="0"
+              max="60000"
+              step="100"
+              value="250000"
+            />
+            <input type="text" name="input_price" id="min_price" />
+            <input type="text" name="input_price" id="max_price" />
+          </fieldset>
+
+          <fieldset>
+            <legend>Производители:</legend>
+            <ul>
+              <li>
+                <input type="checkbox" name="BOSCH" id="filter-BOSCH" />
```

Comment:
```suggestion
                <input type="checkbox" name="bosch" id="filter-bosch" />
```

давай-ка поправь все названия на аккуратные маленькие буквы, и все айдишники тоже

Reply:
Также, посмотри на дизайн, это поле BOSCH - чекнутое, значит нужен тут атрибут, которые бы указывал на это.